docs/UserGuide.md

```diff
@@ -180,6 +208,85 @@ If your changes to the data file makes its format invalid, AddressBook will disc
 Furthermore, certain edits can cause the AddressBook to behave in unexpected ways (e.g., if a value entered is outside of the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 </div>
 
+---
+
+### **Log Interaction History**
+
+Records an interaction or communication with a specific contact, such as calls, meetings, or messages.
+
+**Command Format:**  
+`log <ContactID> [d/<Date>] l/<Interaction Details>`
+
+- `d/` Date of the interaction (optional)
+- `l/` Details of the interaction
+
+**Example:**
+- `log 123 d/2024-09-16 l/Discussed property updates`
+
+---
```

Comment:
Mention that id d/ is absent, the logging date is set to today system time by default